src/alerts.rs

```diff
@@ -100,15 +177,15 @@ impl<H: Hasher, D: Data, KB: KeyBox> Alerter<H, D, KB> {
     fn units_are_legit(
```

Comment:
The term "legit" has some specific meaning (because I said so long ago :) ): roughly speaking legit = "either from a non-forker or from alert". So it would be better to call this function differently. The field in the Alert field is fine though, as it is consistent with the above semantics.

Reply:
Changed to `correct_commitment`.